lib/data/wrapper.js

```diff
@@ -6,6 +6,7 @@ import inMemory from './in_memory/backend';
 import config from '../Config';
 import MD5Sum from '../utilities/MD5Sum';
 import assert from 'assert';
+import DpClient from './dpa/dpClient';
```

Comment:
I think it would be clearer to call this middleware rather than a client because really what it is doing is calling each of the individual backend clients.